include/envoy/event/deferred_task.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
```

Comment:
I don't think this is used in this PR, please remove.

Reply:
Will do